torchaudio/models/wav2vec2/components.py

```diff
@@ -314,7 +318,203 @@ def forward(
         output = output.transpose(2, 1).reshape(batch_size, length, embed_dim)
 
         output = self.out_proj(output)
-        return output
+        return output, None  # Necessary for compatibility with WavLMSelAttention
+
+
+class WavLMSelfAttention(nn.Module):
```

Comment:
So as to be extra careful of the license, can you move this to a separate file, then add license notice from the original unilm, instead of unispeech? We cannot port code from unispeech repo, even though they ported the code from unilm.